loop/routes/rooms.js

```diff
@@ -201,25 +201,27 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
       roomData.updateTime = now;
       roomData.expiresAt = now + roomData.expiresIn * tokenlib.ONE_HOUR;
       roomData.roomOwnerHmac = req.user;
+      roomData.channel = req.body.channel || "default";
```

Comment:
I believe we don't have any requirements so far on this, but it seems to make sense to log this information to extract metrics from it.